snuba/manual_jobs/manifest_reader.py

```diff
@@ -1,18 +1,34 @@
 import os
-from typing import Any, Sequence
+from typing import Mapping
 
 import simplejson
 
+from snuba.manual_jobs import JobSpec
 
-class _ManifestReader:
+
+class ManifestReader:
```

Comment:
is there a reason that you want to make `ManifestReader` "public"? I'd prefer to expose as little as possible.

not a blocker for merge

Reply:
at one point I was using it outside, but then that got deleted and i forgot to change it back to private